ui/app/pages/permissions-connect/choose-account/choose-account.component.js

```diff
@@ -76,31 +125,67 @@ export default class ChooseAccount extends Component {
     )
   }
 
+  renderAccountsListHeader () {
+    const { t } = this.context
+    const { selectNewAccountViaModal } = this.props
+    return (
+      <div className="permissions-connect-choose-account__accounts-list-header">
+        <div className="permissions-connect-choose-account__select-all">
+          <CheckBox
+            className="permissions-connect-choose-account__header-check-box"
+            checked={this.allAreSelected()}
+            onClick={() => (this.allAreSelected() ? this.deSelectAll() : this.selectAll())}
+          />
+          <div className="permissions-connect-choose-account__text-grey">{ this.context.t('selectAll') }</div>
+          <Tooltip
+            position="bottom"
+            html={(
+              <div style={{ width: 129, height: 112, padding: 4 }}>
```

Comment:
Nit: This looks much better without specifying the height. Statically setting the height and width is especially problematic when other languages are set that might be longer.

```suggestion
              <div style={{ width: 129, padding: 4 }}>
```

You might want to consider increasing the width as well? 🤔 I definitely agree we should set _some_ width, but something like 200px is a lot more readable imo, even in the popup UI.

```suggestion
              <div style={{ width: 200, padding: 4 }}>
```

Reply:
This was accidentally included in commit bd8c537cb9, but that is where it will stay :)